src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs

```diff
@@ -471,6 +471,16 @@ public bool ValidateIssuerSigningKey
         }
 
         /// <summary>
+        /// Gets or sets the boolean to control if the signature of the token will be validated during token validation.
+        /// </summary>
+        [DefaultValue(true)]
```

Comment:
I understood it's usage incorrectly. Added tests for both positive and negative cases.